packages/web-components/fast-foundation/src/menu-item/menu-item.template.ts

```diff
@@ -60,16 +56,11 @@ export function menuItemTemplate<T extends FASTMenuItem>(
         ${when(
             x => x.hasSubmenu,
             html<T>`
-                <div
-                    part="expand-collapse-glyph-container"
-                    class="expand-collapse-glyph-container"
-                >
-                    <span part="expand-collapse" class="expand-collapse">
-                        <slot name="expand-collapse-indicator">
-                            ${options.expandCollapseGlyph ?? ""}
-                        </slot>
-                    </span>
-                </div>
+                <span part="submenu-icon" class="submenu-icon">
+                    <slot name="submenu-icon">
+                        ${options.submenuIcon ?? ""}
+                    </slot>
+                </span>
             `
         )}
         ${when(
```

Comment:
Could this be moved inside the `hasSubmenu` template? I
m assuming that `expanded` doesn't matter unless `hasSubmenu` is `true`.